pkg/common/bmetrics/batch.go

```diff
@@ -0,0 +1,73 @@
+package bmetrics
+
+import (
+	"github.com/matrixorigin/matrixone/pkg/util/metric"
+	"go.uber.org/atomic"
```

Comment:
use golang system metric.

Reply:
sure. Golang doesn't provide atomic.Float64. Since `v.globalCounter.Add(float64)` accepts float64, I thought of using atomic.Float64. Will see if I can add atomic.Float64 code to the current codebase.